site/_posts/2019-05-21-Announcing-Hammer-6.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Hammer-6
+author: jprause
+date: 2019-04-23 08:35:39 UTC
```

Comment:
maybe a more recent date? :)

Reply:
Oopsey. Looks like I looked at the month of April on my calendar. :roll_eyes: